server/entity/arrow.go

```diff
@@ -191,6 +191,11 @@ func (a *Arrow) Rotation() (float64, float64) {
 	return a.yaw, a.pitch
 }
 
+// blocker represents an entity that can block attacks with a shield.
+type blocker interface {
+	Blocking() (bool, bool)
```

Comment:
I would document this method and probably add named return types here, because it's unclear what this should return.

Reply:
note: resolved